### PR TITLE
JMX and Actuator port work

### PR DESCRIPTION
**Pull request by mikebell90**

### CODE REVIEW

**Comment by mikebell90** [comment]
No Adam for thise they have to manually change

File: otj-server-core/src/main/java/com/opentable/server/JmxConfiguration.java [hunk 1]
```diff
@@ -41,7 +41,7 @@
 import com.opentable.server.JmxConfiguration.JmxmpServer;
 import com.opentable.service.AppInfo;
 import com.opentable.service.K8sInfo;
-import com.opentable.service.PayloadSelector;
+import com.opentable.service.PortSelector;
```
Comment (strykerd) [inline otj-server-core/src/main/java/com/opentable/server/JmxConfiguration.java:44]: Where are PayloadSelector and PortSelector defined?  I don't see them in this PR, nor in tip master otj-server.

**Comment by mikebell90** [comment]
This will probably be closed in favor of #120 once it is merged into #118

**Comment by mikebell90** [comment]
Made obsolete by Mr Kaukov's work on #118

**Pull request closed without merging**